webapp/src/js/bootstrapper/translator.js

```diff
@@ -8,6 +8,13 @@ var TRANSLATIONS = {
       return 'Fetching info (' + x + ' docs)…';
     },
     LOAD_APP: 'Loading app…',
+    PURGE_INIT: 'Checking data…',
+    PURGE_INFO: function(progress) {
+      const n = progress.purged;
+      const percent = Math.floor((progress.processed / progress.total) * 100);
+      return `Cleaned ${n} documents (${percent}% complete)…`;
```

Comment:
To make translation easier, perhaps do the calculation first, and call the translator with `count` and `percent`. Otherwise each language will have to do this calculation...